contrib/autoname-workspaces.py

```diff
@@ -31,7 +31,7 @@ def icon_for_window(window):
     else:
         # xwayland support
         class_name = window.window_class
-        if len(class_name) > 0:
+        if class_name is not None and len(class_name) > 0:
```

Comment:
Maybe avoid the nested else/if and duplication doing something like:

```python
name = None
if window.app_id is not None and len(window.app_id) > 0:
    name = window.app_id.lower()
elif window.window_class is not None and len(window.window_class) > 0:
    name =  window.window_class.lower()

if name in WINDOW_ICONS:
    return WINDOW_ICONS[name]

logging.info("No icon available for window with name: %s" % str(name))
return DEFAULT_ICON
```

Reply:
Thanks! I tested the refactored function and it worked. :slightly_smiling_face: 

I committed the refactored function to this PR.